VictoriaAlbert/VictoriaAlbert/MuseumObject.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  MuseumObject.swift
+//  VictoriaAlbert
+//
+//  Created by Jermaine Kelly on 11/10/16.
+//  Copyright © 2016 Jermaine Kelly. All rights reserved.
+//
+
+import Foundation
+
+class Record{
+    
+    let name: String
+    let place: String
+    let title: String
+    let date: String
+    let artist: String
+    let location: String
+    let imageID: String
+    
+    
+    init (name:String, place:String, title:String, date:String,artist:String,location:String, imageID:String){
+        self.name = name
+        self.place = place
+        self.title = title
+        self.date = date
+        self.artist = artist
+        self.location = location
+        self.imageID = imageID
+    }
+    
```

Comment:
Don't have more than one blank line; two at most. Whatever you choose, be consistent.